app/src/main/java/com/shub39/rush/lyrics/util.kt

```
@@ -64,6 +64,13 @@ fun getCurrentLyricIndex(playbackPosition: Long, lyrics: List<Lyric>): Int {
     }
 }
 
+fun getNextLyricTime(index: Int, lyrics: List<Lyric>): Long? {
+    for (i in index + 1 until lyrics.size) {
+        return lyrics[i].time
+    }
```

detekt (1.23.8)

[warning] This loop contains an unconditional jump expression which essentially renders it useless as it will exit the loop during the first iteration.

(detekt.potential-bugs.UnconditionalJumpStatementInLoop)